firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/ImagenModel.kt

```diff
@@ -58,6 +85,12 @@ internal constructor(
       throw FirebaseVertexAIException.from(e)
     }
 
+  /**
+   * Generates an image, storing the result in Google Cloud Storage and returning a URL
+   *
+   * @param prompt The input(s) given to the model as a prompt.
+   * @param gcsUri Specifies the GCS bucket in which to store the image.
```

Comment:
```suggestion
   * @param gcsUri Specifies the Google Cloud Storage bucket in which to store the image.
```

Reply:
Is calling it a bucket ok here? This may not be entirely accurate, since it can be a folder in a bucket or any other gs:// location

Reply:
maybe?

Specifies where in Google Cloud Storage to store the image (for example, a specific bucket or folder).